cli/dcoscli/marathon/main.py

```diff
@@ -874,22 +874,40 @@ def task_kill(self, task_ids, scale, wipe):
         :type scale: bool
         :param wipe: whether remove reservations and persistent volumes.
         :type wipe: bool
+        :param json_: output JSON if true
+        :type json_: bool
         :returns: process return code
         :rtype: int
         """
 
         client = self._create_marathon_client()
         payload = client.kill_and_scale_tasks(task_ids, scale, wipe)
 
-        # No deployment was started and no tasks were killed
+        def print_deployment(deployment, json_):
+            if json_:
+                emitter.publish('Created deployment:')
+                emitter.publish(deployment)
+            else:
+                emitter.publish('Created deployment: {}'.format(
+                    deployment['deploymentId']))
+
+        def print_killed_tasks(tasks, json_):
+            if json_:
+                emitter.publish('Killed tasks:')
+                emitter.publish(tasks)
+            else:
+                emitter.publish('Killed tasks: {}'.format(
+                    [task['id'] for task in tasks]))
+
         if scale:
-            emitter.publish("Started deployment: ")
-            emitter.publish(payload)
+            print_deployment(payload, json_)
         else:
-            emitter.publish('Killed tasks: ')
-            emitter.publish(payload['tasks'])
-            if len(payload['tasks']) == 0:
+            tasks = payload['tasks']
```

Comment:
since we're returning json, let's return the response as is (ie remove this line).